src/client/pages/federation.vue

```diff
@@ -28,14 +28,14 @@
 				<option value="-following">{{ $ts.following }} ({{ $ts.ascendingOrder }})</option>
 				<option value="+followers">{{ $ts.followers }} ({{ $ts.descendingOrder }})</option>
 				<option value="-followers">{{ $ts.followers }} ({{ $ts.ascendingOrder }})</option>
-				<option value="+caughtAt">{{ $ts.caughtAt }} ({{ $ts.descendingOrder }})</option>
-				<option value="-caughtAt">{{ $ts.caughtAt }} ({{ $ts.ascendingOrder }})</option>
-				<option value="+lastCommunicatedAt">{{ $ts.lastCommunicatedAt }} ({{ $ts.descendingOrder }})</option>
-				<option value="-lastCommunicatedAt">{{ $ts.lastCommunicatedAt }} ({{ $ts.ascendingOrder }})</option>
+				<option value="+caughtAt">{{ $ts.registeredAt }} ({{ $ts.descendingOrder }})</option>
+				<option value="-caughtAt">{{ $ts.registeredAt }} ({{ $ts.ascendingOrder }})</option>
+				<option value="+lastCommunicatedAt">{{ $ts.lastRequest }} ({{ $ts.descendingOrder }})</option>
+				<option value="-lastCommunicatedAt">{{ $ts.lastRequest }} ({{ $ts.ascendingOrder }})</option>
```

Comment:
lastCommunicatedAtは「直近のリクエスト送信」だけではなく「直近のリクエスト受信」も含まれた時間なので、「直近の通信」がいいと思います。

Since lastCommunicatedAt is the time that includes not only "last request sent" but also "last request received", I think "直近の通信"(last communication) is better.

Reply:
👍  I changed it

Reply:
You've not changed the name of the localization string yet, might also want to do that.